tests/test_scripts/test_mcp_launchers.py

```
@@ -0,0 +1,158 @@
+
+def test_example_machine_env_can_be_sourced_by_bash() -> None:
+    result = subprocess.run(
+        [
+            "bash",
+            "-lc",
+            (
+                "set -euo pipefail; "
+                "source configs/machines/example.env; "
+                'printf \'%s\\n\' "$CHATGPT_TOKENS_PATH" "$GEMINI_COOKIES_PATH"'
+            ),
+        ],
```

Ruff (0.15.7)

[error] Starting a process with a partial executable path

(S607)